File: js/anagram2.js
exports.anagramsFor = function(word, listOfWords) {
    let answerArr = [];
    let sortWord = word.split("").sort().join("");

    for (i=0;i<listOfWords.length;i++) {
       let iSort = listOfWords[i].split("").sort().join("");
       if (sortWord === iSort) {
           answerArr.push(listOfWords[i])
       };
    };
    return answerArr;
};

File: python/anagram2.py
# Don't forget to run the tests (and create some of your own)

def anagrams_for(word, list_of_words):
	answerArr = []
	wordSort = ''.join(sorted(word))

	for list in list_of_words:
		sortList = ''.join(sorted(list))
		if wordSort == sortList:
			answerArr.append(list)
	return answerArr


		# your code here
